modules/install-sno-generate-the-discovery-iso.adoc

```diff
@@ -0,0 +1,51 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="generate-the-discovery-iso_{context}"]
+
+= Generate the discovery ISO
+
+Single node OpenShift (SNO) installs with a discovery ISO, which the Assisted Installer (AI) generates with the cluster name, base domain, SSH public key, and pull secret.
+
+.Procedure
+
+. On the administration node, open a browser and navigate to link:https://cloud.redhat.com/openshift/assisted-installer/clusters[Install OpenShift with the Assisted Installer].
+
+. Click *Create New Cluster* to create a new cluster.
+
+. In the *Cluster Name* field, enter a name for the cluster.
+
+. In the *Base Domain* field, enter a base domain. For example:
++
+----
+example.com
+----
++
+All DNS records must be subdomains of this base domain and include the cluster name. You cannot change the base domain after cluster installation. For example:
++
+----
+<cluster-name>.example.com
+----
+
+. Select *"I want to install single node OpenShift (SNO)"*.
+
+. Read the limitations of SNO.
+
+. Select *"I understand, accept, and agree to the limitations associated with using Single Node OpenShift"*.
+
+. Select the SNO version.
+
+. Click *Next*.
+
+. Click *Generate Discovery ISO*.
+
+. Select *Full ISO* to boot with a USB drive or Portable eXecution Environment (PXE). Select *Minimal ISO* to boot with virtual media.
```

Comment:
nit: "USB drive or PXE."

If the RHEL folks don't have to spell it out, then my the transitive law, neither do we.

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/chap-booting-installer-x86#sect-booting-from-pxe-x86